package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "buying-catalogue-library",
-  "version": "1.3.3",
+  "version": "1.3.4",
```

Comment:
Let's make this a minor update and let's update `package-lock` as well.

Recommend just running `npm version minor --no-git-tag-version` as usual.